public/index.html

```diff
@@ -2,29 +2,17 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="A math game with dinosaurs!"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
```

Comment:
since you added the android ones to the folder. Can we go ahead and add those links also?

```
<link rel="android-chrome-icon-sm" sizes="192x192" href="%PUBLIC_URL%/android-chrome-192x192.png" />
<link rel="android-chrome-icon-lg" sizes="512x512" href="%PUBLIC_URL%/android-chrome-512x512.png" />

```

Just in case something gets messed up for not including %PUBLIC_URL% <-- lets include it unless you have a reason for removing it.

The four link tags you have for line 11-14 are not self closing. You will want to add the slash at the end />

Reply:
Done! I checked on the `rel` attribute. Seems that the best value in this case is just `icon`, so I went with that and added `type="image/png"`, too.